src/Symfony/Component/Intl/Data/Generator/TimezoneDataGenerator.php

```diff
@@ -96,20 +121,63 @@ protected function generateDataForMeta(BundleReaderInterface $reader, $tempDir)
     {
         $rootBundle = $reader->read($tempDir, 'root');
 
-        $this->zoneCodes = array_unique($this->zoneCodes);
+        $this->zoneIds = array_unique($this->zoneIds);
 
-        sort($this->zoneCodes);
+        sort($this->zoneIds);
 
         $data = [
             'Version' => $rootBundle['Version'],
-            'Zones' => $this->zoneCodes,
+            'Zones' => $this->zoneIds,
+            'ZoneToCountry' => self::generateZoneToCountryMapping($reader->read($tempDir, 'windowsZones')),
         ];
 
+        $data['CountryToZone'] = self::generateCountryToZoneMapping($data['ZoneToCountry']);
+
         return $data;
     }
 
-    private static function generateZones(ArrayAccessibleResourceBundle $typeBundle, ArrayAccessibleResourceBundle $metaBundle, ArrayAccessibleResourceBundle $rootBundle, ArrayAccessibleResourceBundle $localeBundle): array
+    private function generateZones(string $locale, ArrayAccessibleResourceBundle $typeBundle, ArrayAccessibleResourceBundle $metaBundle, ArrayAccessibleResourceBundle $windowsZonesBundle, ArrayAccessibleResourceBundle ...$localeBundles): array
     {
+        $accessor = function (ArrayAccessibleResourceBundle $resourceBundle, array $indices) {
```

Comment:
This doesn't use the `$this` scope and can be made `static`.

Reply:
fixed.